stable/redis/values-production.yaml

```diff
@@ -0,0 +1,230 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis
```

Comment:
since this is a major version update, I would recommend updating to the new structure,

```
image:
  registry: docker.co
  repository: bitnami/redis
  tag: 4.0.9-r0
  pullPolicy: IfNotPresent
```